app/views/icons/mini/_magnifying.html.erb

```diff
@@ -0,0 +1,3 @@
+<svg class="w-4 h-4 text-gray-500 dark:text-gray-400" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 20 20">
```

Comment:
Pass in the custom text color through `classes`
```suggestion
<svg class="w-4 h-4 <%= classes %>" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 20 20">
```

Reply:
Only reason it's set that way is because I copy pasted tailwind icon lol. I'll change it.